ground-server/package-lock.json

```diff
@@ -64,7 +73,6 @@
       "resolved": "https://registry.npmjs.org/@eslint/eslintrc/-/eslintrc-2.1.4.tgz",
       "integrity": "sha512-269Z39MS6wVJtsoUl10L60WdkhJVdPG24Q4eZTH3nnF6lpvSShEK3wQjDX9JRWAUPvPh7COouPpU9IrqaZFvtQ==",
       "dev": true,
-      "license": "MIT",
```

Comment:
Very unimportant, but do you know why the MIT licenses were removed here?

Reply:
I don't know, but this file is auto generated by npm. Maybe eslint removed that themselves?